Jenkinsfile

```diff
@@ -2,10 +2,14 @@ pipeline {
   agent {label 'master'}
 
   environment {
-    VERSION = sh(returnStdout: true, script: 'mvn org.apache.maven.plugins:maven-help-plugin:2.1.1:evaluate -Dexpression=project.version | grep -Ev "(^\\[|Download\\w+)"').trim()
     RELEASE_REGEX = /^([0-9]+(\.[0-9]+)*)(-(RC|beta-|alpha-)[0-9]+)?$/
     RELEASE_DEPLOY = false
     SNAPSHOT_DEPLOY = false
+
+    VERSION = sh(returnStdout: true, script: 'mvn org.apache.maven.plugins:maven-help-plugin:2.1.1:evaluate -Dexpression=project.version | grep -Ev "(^\\[|Download\\w+)"').trim()
+    PACKAGING = sh(returnStdout: true, script: 'mvn org.apache.maven.plugins:maven-help-plugin:2.1.1:evaluate -Dexpression=project.packaging | grep -Ev "(^\\[|Download\\w+)"').trim()
```

Comment:
are these variables used anywhere?

Reply:
`VERSION` yes, the other were used in a former test version of this implementation. Thought it could be useful for further changes in the Jenkinsfile, since they take almost no time. If you want, I can remove them.

Reply:
no, I don't think it is necessary, we might actually use them later